os/pm/pm_test.c

```diff
@@ -47,53 +47,12 @@ int pmtest_dev_prepare(struct pm_callback_s *cb, int domain, enum pm_state_e sta
 	return OK;
 }
 
-/* PM test routine */
-
-static int pmtest_kthread(int argc, char *argv[])
-{
-	enum pm_state_e s = PM_IDLE;
-#ifdef CONFIG_PM_METRICS
-	struct pm_time_in_each_s m;
-#endif
-	struct timespec start_time;
-	struct timespec elapsed_time;
-
-	pmvdbg("This test runs for 7200 secs\n");
-	clock_gettime(CLOCK_REALTIME, &start_time);
-	elapsed_time.tv_sec = start_time.tv_sec;
-
-	while (elapsed_time.tv_sec <= (start_time.tv_sec + PMTEST_DURATION_IN_SECS)) {
-		sleep(PMTEST_THREAD_SLEEP_TIME);
-		pm_changestate(PMTEST_DOMAIN, s);
-		pm_dumpstates();
-#ifdef CONFIG_PM_METRICS
-		pm_get_domainmetrics(0, &m);
-		pmvdbg("Normal:%d Idle:%d standby:%d sleep:%d \n", m.normal, m.idle, m.standby, m.sleep);
-#endif
-		clock_gettime(CLOCK_REALTIME, &elapsed_time);
-	}
-	return 0;
-}
-
-/* Launch pm test thread */
-
-void pmtest_launch_kthread(void)
-{
-	int i = 0;
-	if (kernel_thread("pmtest", PMTEST_THREAD_PRIORITY, PMTEST_THREAD_STACKSIZE, pmtest_kthread, (char *const *)NULL) < 0) {
-		pmvdbg("pmtest kthread launch failed\n");
-		for (i = 0; i < PMTEST_DEVICES; i++) {
-			pm_unregister(&pmtest_cbarray[i]);
-		}
-	}
-}
-
 /* init function  */
 
 void pmtest_init(void)
 {
 	int i = 0;
-	char pmtest_dev_names[PMTEST_DEVICES][CONFIG_PM_DEVNAME_LEN] = { "dev0", "dev1", "dev2" };
+	char pmtest_dev_names[PMTEST_DEVICES][CONFIG_PM_DEVNAME_LEN] = { "app0", "app1", "app2" };
```

Comment:
This change can be added to a different commit.

Reply:
done